src/ckb/channel.rs

```diff
@@ -3706,24 +3689,14 @@ impl ChannelActorState {
         // for xudt compatibility issue,
         // refer to: https://github.com/nervosnetwork/cfn-scripts/pull/5
         let empty_witness_args: [u8; 16] = [16, 0, 0, 0, 16, 0, 0, 0, 16, 0, 0, 0, 16, 0, 0, 0];
-        let witnesses: Vec<u8> = [
```

Comment:
The content of witness is quite useful while debugging. Can we add a `trace!` here?

Reply:
there is a debug level log in the fn build_commitment_transaction_parameters already, it's redundant

```rust
        let witnesses: Vec<u8> = self.build_current_commitment_transaction_witnesses(local);

        let hash = blake2b_256(&witnesses);
        let script_arg: &[u8] = &hash[..20];
        debug!(
            "Building {} commitment transaction with witnesses {:?} and hash {:?} with local commitment number {} and remote commitment number {}",
            if local { "local" } else {"remote"},
            hex::encode(&witnesses),

```